var/spack/repos/builtin/packages/sperr/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Sperr(CMakePackage):
+    """SPERR is a lossy scientific (floating-point) data compressor that can
+    perform either error-bounded or size-bounded data compression"""
+
+    homepage = "https://github.com/shaomeng/SPERR"
+    git = "https://github.com/robertu94/SPERR"
+
+    version("2022.07.18", commit="640305d049db9e9651ebdd773e6936e2c028ff3a")
+    version("2022.05.26", commit="7894a5fe1b5ca5a4aaa952d1779dfc31fd741243")
+
+    depends_on("git", type="build")
+    depends_on("zstd", type=("build", "link"), when="+zstd")
+    depends_on("pkgconf", type=("build"), when="+zstd")
+
+    variant("shared", description="build shared libaries", default=True)
+    variant("qz", description="coding terminates by quantization level", default=True)
+    variant("zstd", description="use Zstd for more compression", default=True)
+
+    maintainers = ["robertu94"]
+
+    def cmake_args(self):
+        args = [
+            self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
+            self.define_from_variant("QZ_TERM", "qz"),
```

Comment:
Similarly, this option can be removed. 

However, this is an option `USE_OMP` which if not specified, will be turned on by default. I'm not sure if you want to include that one or not.

Reply:
What would you want the default to be? I would assume on?

Reply:
The CMake option, `QZ_TERM`, is no longer there, so we should completely delete it. 

I was suggesting that another CMake option, `USE_OMP`, should be set `ON` if possible ;-)